README.md

```diff
@@ -143,13 +143,23 @@ password in postgres to `postgres`.
 
 
 ### Run tests
-    (conbench) $ cd ~/workspace/conbench/
-    (conbench) $ pytest -vv conbench/tests/
```

Comment:
Notably, this method is not easily expected to succeed because it does not automatically manage Dex.